### ISSUE TITLE
Login to existing account on another machine, without backup

### ISSUE THREAD

**Issue by asengineeringsupport** [qTox/qTox#3501]
Hello everyone,
I'm running some tests for implementation qTOX in an enterprise environment.
I know that's possible to backup and restore profiles, but it is possible to login to an existing account on a another machine, without a backup? I doens't need my old avatar, or contact list. I'll do it only for installation and update purposes on several machines.

One example: The endusers doesn't have administrative permission to install the qTOX Client, so i need to install it with my admin account. I went to a new machine, installed the software and updated it with my account in the past. I found the qtox-updater.exe in installation destination path, but it crashes when I run in manually. So I need to create a new account, wait until the update notification appears and confirm the installation. After this I log off from computer, the enduser logs in, creates also a new accound and begins to work.

I doesn't want to create every time a new account to install and update the software as Administrator.

Is there a possibility to login with my existing account, when I know my TOX-ID and the Password, without restore the complete profile from another machine.?

Best regards,
Peter

**Comment by zetok** [comment]
> Is there a possibility to login with my existing account, when I know my TOX-ID and the Password, without restore the complete profile from another machine.?

Not quite. Profile data is only in files – if you don't have that data, you basically don't have the profile. Password is needed only for encrypting/decrypting profile data when written/read from storage, and qTox can work without it, although then you don't have encrypted data anymore.

Your Tox ID has only part of needed data – public key, which can be freely shared with anyone. What really makes your profile, is secret key, which is only located in a single file, named `<profile_name>.tox`.

So, the "profile" data exists only locally, in `\%APPDATA%\tox\`. Thus "remote" login is not an option, unless `\%APPDATA%\tox\` would be a mounted remote share, or whatever windows can use for it. There was some mechanism that (user directory, etc?) could be mounted in windows from remote server, I don't remember name of it though.

Basically, you need those files to be available in this location in order for users to load their profile.

https://github.com/tux3/qTox/wiki/Profile

**Comment by sudden6** [comment]
>   I found the qtox-updater.exe in installation destination path, but it crashes when I run in manually.

I think the qtox-updater.exe shouln't crash even if there's no profile.

Are you running the latest build of qTox?

Did you try a nightly version from https://build.tox.chat/view/qtox/job/qTox_pkg_windows_x86-64_nightly_release/ ? (for testing purpose only)

**Comment by asengineeringsupport** [comment]
Hello,

I've  downloaded the Version under this link.
https://build.tox.chat/view/qtox/job/qTox_pkg_windows_x86-64_stable_release/lastSuccessfulBuild/artifact/setup-qtox.exe

This version updates automatically to v1.4.1-1-g06d78dc, when you confirm the query.

I think all my questions have been answered. Thank you for your help!

Best regards,
Peter

**Comment by Jackgi** [comment]
So how im'a suppose to migrate from mac to windows ?

**Comment by sudden6** [comment]
@Jackgi 
copy over the `tox` directory, not sure where exactly it's located on mac. On Linux it's `~/.config/tox`.
